java/registry/src/main/java/io/opensaber/registry/shard/advisory/AdvisoryLoader.java

```diff
@@ -0,0 +1,32 @@
+package io.opensaber.registry.shard.advisory;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class AdvisoryLoader {
```

Comment:
This name sounds generic. Let us call it ShardAdvisor and think of making the getShard(String property) as a static method. We expect this class to be initiated once.

Reply:
Named it as  ShardAdvisorLoader:getShardAdvisory(String property)

Reply:
getShardAdvisor() please...

Reply:
Right, done.